src/localization/en-us/commands.json

```diff
@@ -346,6 +346,9 @@
     "suggestionRejectedColor": "0xcc0000",
     "suggestionRejectedTitle": "Suggestion Rejected.",
     "suggestionRejectedDescription": "A suggestion sent by {user-mention} was rejected.",
+    "suggestionImplementedColor": "1030B0",
+    "suggestionImplementedTitle": "Suggestion Designated As Implemented.",
+    "suggestionImplementedDescription": "A suggestion sent by {user-mention} was designated as implemented.",
```

Comment:
I'd prefer the text "Suggestion Already Implemented." as the title and "A suggestion sent by {user-mention} was already implemented." for the description.